vispy/testing/_testing.py

```diff
@@ -408,9 +409,13 @@ def run_tests_if_main():
     except Exception:
         pass
     import __main__
-    print('==== Running tests in script\n==== %s' % fname)
-    run_tests_in_object(__main__)
-    print('==== Tests pass')
```

Comment:
This seems like it should work okay. I assume the advantage is more informative output on error?